protocols/peer/contracts/Peer.sol

```diff
@@ -215,6 +229,9 @@ contract Peer is IPeer, Ownable {
     require(_order.maker.wallet == msg.sender,
       "MAKER_MUST_BE_SENDER");
 
+    require(_order.taker.wallet == tradeWallet || _order.taker.wallet == address(0),
```

Comment:
how is it possible that _order.taker.wallet == address(0)?
if _order.taker.wallet is 0x0, then Swap.swap, sets the takerWallet to the Peer, however, the Peer doesn't have any funds nor can the owner withdraw tokens from it meaning the swap cannot succeed.

Reply:
Discussed in person and removed